bin/nipype_crash_search

```diff
@@ -1,11 +1,12 @@
-#!/usr/bin/env python
+#!python
```

Comment:
"#!python" is being used in the other scripts, e.g., nipype_display_crash.
I tought about it and I am not sure if "#!/usr/bin/env python" works on Windows (if there is any Windows user here).